docs/devctr-image.md

```diff
@@ -46,9 +46,14 @@ registry. The Firecracker CI suite must also be updated to use the new image.
 
 1. Build a new container image with the updated Dockerfile.
 
+   Run the following command.
```

Comment:
This line is not consistent with the one below. We should remove it and just end the sentence above with a `:`

Reply:
made some edits. let me know if it looks better. Happy to reiterate.